src/applications/vaos/tests/containers/VAOSApp.unit.spec.jsx

```diff
@@ -144,7 +144,7 @@ describe('VAOS <VAOSApp>', () => {
     expect(window.location.replace.firstCall.args[0]).to.contain('verify');
     expect(tree.find('RequiredLoginView').exists()).to.be.true;
 
-    window.location.replace = oldLocation;
+    window.location = oldLocation;
```

Comment:
Do the tests pass without this change? I don't have any objection to this change. I'm really just curios. 

My understanding of the difference between the `replace` method and mutating the `location` value is related to the browser history. 

Source: https://stackoverflow.com/a/1865840/3343716

Reply:
yeah they do. this "fix" is really not that important. With the old way `window.location` was never properly restored. but that really doesn't matter because any test that need to spy on `window.location` methods are going to re-mock it anyway.